src/api/app/assets/javascripts/webui/application/request.js

```diff
@@ -203,7 +203,7 @@ function setupActionLink() { // jshint ignore:line
     $('#action_display_' + index).show();
     // It is necessary to refresh the CodeMirror editors after switching tabs to initialise the dimensions again.
     // Otherwise the editors are empty after calling show().
-    editors.forEach( function(editor) { editor.refresh(); });
+    editors.forEach( function(editor) { editor.refresh(); cmMarkDiffLines(); }); // jshint ignore:line
```

Comment:
why do you need to ignore the linter?

Reply:
> why do you need to ignore the linter?

Otherwise I get an `'cmMarkDiffLines' is not defined` error with jshint, as written in the commit message.

The linter is defined to run in the dev.rake file this way: `jshint ./app/assets/javascripts/`. And `cmMarkDiffLines` is defined in `./app/assets/javascripts/webui2/cm2/use-codemirror.js`. Is it maybe a jshint false error?